crates/relayer/relayer-components/src/relay/impls/packet_relayers/ack/base_ack_packet.rs

```diff
@@ -28,20 +33,34 @@ where
     Relay: HasSourceTargetChainTypes
         + HasRelayClientIds
         + CanRaiseRelayChainErrors
-        + CanSendSingleIbcMessage<MainSink, SourceTarget>,
+        + CanSendSingleIbcMessage<MainSink, SourceTarget>
+        + HasLogger,
     Relay::SrcChain: CanQueryClientStateWithLatestHeight<Relay::DstChain>
         + CanBuildAckPacketMessage<Relay::DstChain>
-        + HasOutgoingPacketType<Relay::DstChain>,
+        + HasOutgoingPacketType<Relay::DstChain>
+        + HasChainId,
     Relay::DstChain: HasClientStateType<Relay::SrcChain>
         + CanBuildAckPacketPayload<Relay::SrcChain>
         + HasWriteAckEvent<Relay::SrcChain>,
+    Relay::Logger: CanLog<LevelInfo>,
 {
     async fn relay_ack_packet(
         relay: &Relay,
         destination_height: &HeightOf<Relay::DstChain>,
         packet: &PacketOf<Relay>,
         ack: &AcknowledgementOf<Relay::DstChain, Relay::SrcChain>,
     ) -> Result<(), Relay::Error> {
+        relay
```

Comment:
We already have some of the logs done in `FullCycleRelayer`. Also, the progress should be at level debug or trace, as many packets are involved. Currently, the log levels in hermes-tracing-logging-components are mostly at trace level.

Reply:
Ok this makes sense, I opened this issue https://github.com/informalsystems/hermes-sdk/issues/552 to define the logging strategy.
I will remove these logs and see if there are steps which should be moved to levels higher than `trace`